### PR TITLE
FOUR-13162: The categories are not listed alphabetically in Launch Pad 

### PR DESCRIPTION
**Pull request by pmPaulis**
## Issue & Reproduction Steps
The categories are not listed alphabetically in Launch Pad 

## How to Test

1. Create categories related to the process
2. Go to Process catalogue
3. See the list of categories
4. 

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13162

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-f5d524c897.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

File: resources/js/processes-catalogue/components/ProcessesCatalogue.vue
Comment (luNunezProcessmaker) [inline resources/js/processes-catalogue/components/ProcessesCatalogue.vue:1]: Delete the line 19
":showDefaultCategory="showDefaultCategory"

Comment (luNunezProcessmaker) [reply resources/js/processes-catalogue/components/ProcessesCatalogue.vue:1]: Line 96:
    "showDefaultCategory: false,"

Comment (luNunezProcessmaker) [reply resources/js/processes-catalogue/components/ProcessesCatalogue.vue:1]: line 101:
   " this.showDefaultCategory = true;"

Comment (luNunezProcessmaker) [reply resources/js/processes-catalogue/components/ProcessesCatalogue.vue:1]: line 154:
"this.showDefaultCategory = false;"

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-f5d524c897.engk8s.processmaker.net

**Pull request merged**